ScoutSuite/providers/gcp/rules/findings/computeengine-vpc-flow-logs-disabled.json

```diff
@@ -0,0 +1,15 @@
+{
+    "description": "VPC Flow Logs Not Enabled",
+    "rationale": "VPC Flow Logs were not enabled for this subnet. It is best practice to enable Flow Logs to some degree in order to have network visibility in the event of resource compromise, as well as source data for threat detections.",
+    "dashboard_name": "Subnetwork",
+    "path": "computeengine.projects.id.regions.id.subnetworks.id",
+    "conditions": [
+        "and",
+        [
+            "computeengine.projects.id.regions.id.subnetworks.id.flowlogs_enabled",
+            "false",
+            ""
+        ]
+    ],
+    "id_suffix": "name"
```

Comment:
The id_suffix should be flowlogs_enabled, if you want the bool value to be highlighted yellow in the partial.